cli/command/manifest/annotate.go

```diff
@@ -0,0 +1,128 @@
+package manifest
+
+import (
+	"fmt"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/distribution/reference"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type annotateOptions struct {
+	target      string // the target manifest list name (also transaction ID)
+	image       string // the manifest to annotate within the list
+	variant     string // an architecture variant
+	os          string
+	arch        string
+	cpuFeatures []string
```

Comment:
CPU features are being removed from OCI and we are going to deprecate them in docker's format. Let's remove them here.